tests/test_expected.py

```diff
@@ -257,6 +257,24 @@ def test_expected_cis(request):
         chunksize=chunksize,
         ignore_diags=ignore_diags,
     )
+
+    # check column names
+    assert list(res_symm.columns) == [
```

Comment:
maybe there should be a test for the expected return columns when clr_weight_name=None?